src/actions/Clear.ts

```diff
@@ -1,26 +1,20 @@
-import {
-  Action,
-  ActionPreferences,
-  ActionReturnValue,
-  Graph,
-  TypedSelection,
-} from "../typings/Types";
-import { ensureSingleEditor } from "../util/targetUtils";
+import { toRawTarget } from "../processTargets/modifiers/RawSelectionStage";
+import { Target } from "../typings/target.types";
+import { Graph } from "../typings/Types";
 import { setSelectionsAndFocusEditor } from "../util/setSelectionsAndFocusEditor";
+import { ensureSingleEditor } from "../util/targetUtils";
+import { Action, ActionReturnValue } from "./actions.types";
 
 export default class Clear implements Action {
-  getTargetPreferences: () => ActionPreferences[] = () => [
-    { insideOutsideType: "inside" },
-  ];
-
   constructor(private graph: Graph) {
     this.run = this.run.bind(this);
   }
 
-  async run([targets]: [TypedSelection[]]): Promise<ActionReturnValue> {
+  async run([targets]: [Target[]]): Promise<ActionReturnValue> {
     const editor = ensureSingleEditor(targets);
+    const rawTargets = targets.map(toRawTarget);
```

Comment:
We probably don't want raw targets, right?  We might want to remove that utility function as well so people aren't tempted to use raw targets for this kinda thing

Reply:
Why not? What matters here is that we don't have any removal delimiters. You can definitely switch it to a plain target if you prefer that.